ssh/ssh_runner.py

```diff
@@ -103,7 +103,7 @@ def _get_base_args(self, bin_name, host):
         return shared_opts
 
     @asyncio.coroutine
-    def run_cmd_return_dict_async(self, cmd, host, namespace, future):
+    def run_cmd_return_dict_async(self, cmd, host, namespace, future, comment):
```

Comment:
Why does the parameter `comment` become the field `stage`? Please make them consistent, probably using `stage` everywhere if that's what the API in the json should be.